var/spack/repos/builtin/packages/gobject-introspection/package.py

```diff
@@ -86,6 +88,12 @@ class GobjectIntrospection(MesonPackage, AutotoolsPackage):
         when="@:1.63.1",
     )
 
+    conflicts(
+        "^python@3.11:",
+        when="@:2.60",
+        msg="Versions prior to 2.72 not compatible with Python 3.11 or later.",
+    )
```

Comment:
This doesn't look correct:
1. There are no 2.x versions
2. The check is for "up to x.60" while the message says "before x.72"

Reply:
Oh shoot! I got my requests crossed here. Addressing this now and double-checking what actually changed at the 1.60.x release vs. the 1.72.x release.

Reply:
Fixed this and improved the message. Thanks for the review! Also clarified some details around the sbang patch and standardized the glib dependency ranges.